src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -156,7 +156,29 @@ public void testNoConflictPG() throws IOException {
         //output has both GATK PrintReads and GATK PrintReads.1 in headers
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads"));
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads.1"));
+    }
+
+    @Test
+    public void testPGOnByDefault() throws IOException {
+        final File inFile = new File(TEST_DATA_DIR, "print_reads_withPG.sam");
+        final File outFile = BaseTest.createTempFile("testNoConflictRG", ".sam");
```

Comment:
Don't use sam files for tests -- use bams, which exercise very different code paths.